components/Modal.js

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect, useState } from "react";
+import ReactDOM from "react-dom";
+import styled from "styled-components";
+
+export default function Modal({ show, children }) {
+  const [isBrowser, setIsBrowser] = useState(false);
+
+  useEffect(() => {
+    setIsBrowser(true);
+  }, []);
```

Comment:
please use the dynamic import functionality from next.js you can read about it in their docs.

Reply:
Ah i forgot that, i will change that! :)